lib/server.js

```diff
@@ -17,7 +17,10 @@ function Server(options) {
     EventEmitter.call(this, options);
 
     this.server = dgram.createSocket('udp4');
-    this.server.on('message', parser.parse.bind(this));
```

Comment:
Not intentional - probabbly just my editor, I'll update and resend.

On Fri, 15 Jan 2016, 03:16 Andrew Paprocki <notifications@github.com> wrote:

> In lib/server.js
> <https://github.com/apaprocki/node-dhcpjs/pull/6#discussion_r49817191>:
>
> > @@ -17,7 +17,10 @@ function Server(options) {
> >      EventEmitter.call(this, options);
> >
> >      this.server = dgram.createSocket('udp4');
> > -    this.server.on('message', parser.parse.bind(this));
>
> Also here just put a space between ){ to keep the style consistent and
> I'll merge it in asap. Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apaprocki/node-dhcpjs/pull/6/files#r49817191>.
>